include/RAJA/policy/cuda/MemUtils_CUDA.hpp

```diff
@@ -275,10 +295,30 @@ bool setupReducers() { return detail::tl_status.setup_reducers; }
 RAJA_INLINE
 cuda_dim_t currentGridDim() { return detail::tl_status.gridDim; }
 
+//! get grid size of current launch
+RAJA_INLINE
+cuda_dim_member_t currentGridSize() { return detail::tl_status.gridDim.x *
+                                             detail::tl_status.gridDim.y *
+                                             detail::tl_status.gridDim.z; }
+
 //! get blockDim of current launch
 RAJA_INLINE
 cuda_dim_t currentBlockDim() { return detail::tl_status.blockDim; }
 
+//! get block size of current launch
+RAJA_INLINE
+cuda_dim_member_t currentBlockSize() { return detail::tl_status.blockDim.x *
+                                              detail::tl_status.blockDim.y *
+                                              detail::tl_status.blockDim.z; }
+
+//! get dynamic shared memory for current launch, you may add to this
```

Comment:
This comment is a little confusing to me. Can you reword it and avoid the run-on sentence?

Reply:
It is confusing to me also.